other/materials_designer/create_cutout_box.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>What is this code - is this for a plot? Save in "utils"/"scripts" and call appropriately</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I think it's a leftover from creation of that cube graphic. Commited by an error. </p>